### PR TITLE
Fix solution manager UI to show install/ update options in detail view

### PR DESCRIPTION
**Pull request by jainaashish**
This was regressed because of LSL fix to make nuget projects initliazed lazy after showing manager ui for the solution. Because of that, it skipped setting `ShowClassicOptions` flag in `Options` class.

Fixes 513646

@rrelyea

### CODE REVIEW

**Comment by nkolev92** [comment]
Most of a revert of https://github.com/NuGet/NuGet.Client/commit/b892fe82d3435ac5fcae0ec664efc4f40dd0bcb7#diff-34b0ffac355671dc8ef37739eefcf328, from what I could tell.

**Review by nkolev92 — APPROVED** [review]

**Review by mishra14 — APPROVED** [review]

**Comment by mishra14** [comment]
@jainaashish Your branch has a [build error](https://devdiv.visualstudio.com/DevDiv/_build/index?buildId=1120228) - 
```

 C:\Windows\Microsoft.NET\Framework\v4.0.30319\Microsoft.WinFx.targets(268,9): error BG1002: File 'Xamls\PackagesErrorBar.xaml' cannot be found. [E:\A\_work\16\s\src\NuGet.Clients\NuGet.PackageManagement.UI\NuGet.PackageManagement.UI.csproj]
2017-11-07T23:10:07.4553508Z   C:\Windows\Microsoft.NET\Framework\v4.0.30319\Microsoft.WinFx.targets(268,9): error BG1003: The project file contains a property value that is not valid. [E:\A\_work\16\s\src\NuGet.Clients\NuGet.PackageManagement.UI\NuGet.PackageManagement.UI.csproj]
```

Please fix that before merging.

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackagesErrorBar.xaml [hunk 1]
```diff
@@ -1,24 +0,0 @@
-<UserControl x:Class="NuGet.PackageManagement.UI.PackagesErrorBar"
```
Comment (mishra14) [inline src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackagesErrorBar.xaml:1]: Removing this file has caused a build error [here](https://github.com/NuGet/NuGet.Client/blob/6fcbd086faa7013b9b09e5b0f36e78059d1198bc/src/NuGet.Clients/NuGet.PackageManagement.UI/NuGet.PackageManagement.UI.csproj#L114). Please fix that.

**Review by mishra14 — APPROVED** [review]
Fix the build issue then 🚢

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackagesErrorBar.xaml (hunk 1 above)
Comment (jainaashish) [reply src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackagesErrorBar.xaml:1]: yeah saw that after the push, need to remove this file from csproj

**Review by emgarten — APPROVED** [review]

**Pull request merged**